Artiom/src/main/java/Retailer.java

```diff
@@ -0,0 +1,65 @@
+
+import java.util.*;
+
+public class Retailer {
+    public static void main(String[] args) {
+
+        //Create first order
+        Map<String, Address> firstUserAddress = new HashMap<>();
+        firstUserAddress.put("Home", new Address("Moldova", "Chisinau", "Stefan cel Mare"));
+        User firstUser = new User("Pikachu", firstUserAddress);
+        List<Item> firstItemList = new ArrayList<>();
+        firstItemList.add(new Item("iPhone", Type.GADGET));
+        Order firstOrder = new Order(firstUser, firstItemList);
+
+        //Create second order
+        Map<String, Address> secondUserAddress = new HashMap<>();
+        secondUserAddress.put("Home", new Address("Ukraine", "Kyiv", "Verkhnia Street"));
+        User secondUser = new User("Slowpoke", secondUserAddress);
+        List<Item> secondItemList = new ArrayList<>();
+        Order secondOrder = new Order(secondUser, secondItemList);
```

Comment:
It seems that you are trying to create an Order with an empty item list. Why not handle this in another constructor, where you wouldn't pass item list parameter and initialize there intemList as an empty list?

Reply:
done